packages/cells/src/widget.ts

```diff
@@ -148,6 +156,11 @@ const DEFAULT_MARKDOWN_TEXT = 'Type Markdown and LaTeX: $ α^2 $';
  */
 const RENDER_TIMEOUT = 1000;
 
+/**
+ * The mime type for attachments
+ */
+const ATTACHMENTS_MIME = 'application/vnd.jupyter.attachments';
```

Comment:
Will you add a parameter string like we discussed? If so, the receiver code should probably handle both cases (with parameter string, and without; if it has it, it tries to call it).

Reply:
This is done